.test-infra/jenkins/Kubernetes.groovy

```diff
@@ -132,4 +132,28 @@ class Kubernetes {
       }
     }
   }
+
+  /**
+   * Specifies steps that will return an available port on Kubernetes cluster
+   *
+   * @param lowRangePort - low range port to be used
+   * @param highRangePort - high range port to be used
+   * @param referenceName - name of the environment variable
+   */
+  void availablePort(String lowRangePort, String highRangePort, String referenceName) {
```

Comment:
is there a way to log what's happening so we can debug in case of failures?

Reply:
This function already outputs to the log console https://ci-beam.apache.org/job/beam_PerformanceTests_Kafka_IO/2789/consoleFull
`13:01:36 + getAvailablePort 32400 32767
13:01:36 + local lowRangePort=32401
13:01:36 + local highRangePort=32767
13:01:36 + local used=false
13:01:36 + local 'command=kubectl --kubeconfig=/home/jenkins/jenkins-slave/workspace/beam_PerformanceTests_Kafka_IO/config-beam-performancetests-kafka-io-2789 --namespace=beam-performancetests-kafka-io-2789 get svc --all-namespaces -o   go-template='\''{{range .items}}{{range.spec.ports}}{{if .nodePort}}{{.nodePort}}{{"\n"}}{{end}}{{end}}{{end}}'\'''`